docs/instances.md

```diff
@@ -170,6 +170,32 @@ User.bulkCreate([
 })
 ```
 
+Insert several rows and return all columns:
+
+```js
+User.bulkCreate([
+  { username: 'barfooz', isAdmin: true },
+  { username: 'foo', isAdmin: true },
+  { username: 'bar', isAdmin: false }
+], { returning: true }) // will return all columns for each row inserted
+.then((result) => {
+  console.log(result);
+});
+```
+
+Insert several rows and return specific columns:
+
+```js
+User.bulkCreate([
+  { username: 'barfooz', isAdmin: true },
+  { username: 'foo', isAdmin: true },
+  { username: 'bar', isAdmin: false }
+], { returning: ['username'] }) // will return only the specified columns for each row inserted
+.then((result) => {
+  console.log(result);
+});
+```
+
```

Comment:
Excellent addition, please just clarify that this is available for postgres only